### ISSUE TITLE
Quick Export Enhancement

### ISSUE THREAD

**Issue by theamanjs** [shaneapen/WhatsApp-Group-Contacts-Scraper#6]
When we run this extension using quick export. Though it runs real fast in the name section it also shows the number. It should show some strings which we can use just like appending the prefix in extension More Options. This can be done by having one more input block in extension more options form. I would like to work on this feature if you say. @shaneapen

**Comment by shaneapen** [comment]
But you could already use the `prefix` option for that, huh? Or are you talking about displaying a `string + increment` such as `Contact 1`, `Contact 2`, etc.?

**Comment by theamanjs** [comment]
Yes, I'm talking about having `string + increment`. That would be easier to handle by user.

**Comment by shaneapen** [comment]
That's a nice feature to have but I doubt whether many people would be needing it anyways. So I think it's better to extend this feature into the `prefix` field itself rather than adding a new field for it. 
The `prefix` field could accept variables in the format say `$num` or `$group-name` which would then be dynamically replaced. A short note about it can be added there as well. Just my thoughts. Feel free to correct me.

**Comment by theamanjs** [comment]
Yeah, sounds great. Just for confirmation `prefix + increment` it would be or it will contain group name as well?

**Comment by shaneapen** [comment]
`Increment` alone would suffice I guess. You can implement it either as a variable insertion into the prefix box as discussed before or through a checkbox toggle.

**Comment by theamanjs** [comment]
Okay, Can I work on this some days later? This will be great if this is possible.

**Comment by shaneapen** [comment]
yeah go ahead
